src/frontend/components/administration/apikey/ApikeyTab.vue

```diff
@@ -1,34 +1,26 @@
 <template>
   <v-card>
     <v-toolbar color="secondary" dark flat dense>
-      <v-toolbar-title v-text="$t('administration.apikey.title')" />
+      <v-toolbar-title> {{ $t('administration.apikey.title') }} </v-toolbar-title>
       <v-spacer />
       <v-icon>mdi-security</v-icon>
     </v-toolbar>
     <v-card-actions>
       <v-spacer />
       <v-btn
         @click.stop="setVisible(true)"
-        v-text="$t('administration.apikey.buttonCreate')"
-      />
-
+      >
+        {{ $t('administration.apikey.buttonCreate') }}
+      </v-btn>
       <CreateDialog
         v-model="createDialogVisible"
         @created="getApikeys()"
         @closed="setVisible(false)"
       />
     </v-card-actions>
 
-    <v-row
-      v-if="apikeys.length === 0"
-      align="center"
-      justify="center"
-      class="ma-2"
-    >
-      <v-col class="text-center" cols="12" sm="4">
-        {{ $t("administration.apikey.noApikeys") }}
-      </v-col>
-    </v-row>
+    <Loader v-if="loading" />
+    <NoData v-else-if="apikeys.length === 0" :local-key="'administration.apikey.noApikeys'" />
```

Comment:
`apikeys` may be undefined

```suggestion
    <NoData v-else-if="!apikeys?.length" :local-key="'administration.apikey.noApikeys'" />
```

Reply:
You can't use "?." in template

Reply:
Ahh dependencies were not updated for quite a while...

It's supported since [Vue2.7](https://github.com/vuejs/vue/blob/main/CHANGELOG.md#270-2022-07-01) :

> 2.7 also supports using ESNext syntax in template expressions. When using a build system, the compiled template render function will go through the same loaders / plugins configured for normal JavaScript. This means if you have configured Babel for .js files, it will also apply to the expressions in your SFC templates.

[Nuxt2.16](https://github.com/nuxt/nuxt/releases/tag/v2.16.0) supports out of the box Vue2.7 (+ it has better support for Node16+, so we can drop `export NODE_OPTIONS=--openssl-legacy-provider` before running `nuxt`)

I suggest that you upgrade deps in a new PR tho

Reply:
And even if `?.` is not supported in template, it doesn't change the fact that `apikeys` can be undefined, so it will crash wen accessing `apikeys.length`